ushahidi/platform-client
# 443 message threads

This pull request makes the following changes:
- Shows email/ SMS messages related to a post.
- Allow a user to reply to an incoming email/ SMS.

Test these changes by:
- pull email/ sms messages using `bin/ushahidi dataprovider incoming`
- Go to the Post detail of a new post created from a message and follow the link view messages associated with the post.
- You should be able to reply to messages and view them on a thread.

Fixes ushahidi/platform#443 .

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/146)
